Expensify/App
# Fix: Blurred images in attachment view on mobile devices

### Details

This PR fixes issue an issue with image attachment view, where the images were blurred out once loaded. 

### Fixed Issues

$ https://github.com/Expensify/App/issues/5193

### Tests


1. Navigate to a conversation
2. Send an image
3. Tap on the image and zoom
4. Verify that the image shown is sharp and fits to screen width.

### QA Steps

1. Navigate to a conversation
2. Send an image
3. Tap on the image and zoom
4. Verify that the image shown is sharp and fits to screen.
5. Repeat above steps with images of different dimensions, especially large images like [this](https://user-images.githubusercontent.com/44479856/132889916-e46ac3dc-19ad-4cca-a033-ec13b8ac79ad.jpg)

### Tested On


### Screenshots


Issue is present only on mobile devices.

#### iOS

![image](https://user-images.githubusercontent.com/6880914/143114882-a110509d-1d48-44ae-af07-9de27d76c4d6.png)



https://user-images.githubusercontent.com/6880914/143115542-1fe0637c-171e-45ac-b94b-ea39eea84380.mp4

#### Android


![Screenshot_1637700815](https://user-images.githubusercontent.com/6880914/143116607-05853edc-f153-4a5c-b085-ce8150acdbc1.png)


https://user-images.githubusercontent.com/6880914/143119438-d12e7d1d-0623-41ab-9f1d-920abbe8ca84.mp4
